include/z64cutscene_commands.h

```diff
@@ -11,445 +11,271 @@
-
-/**
- * ARGS
- *   s8 continueFlag (c), s8 roll (r), s16 frame (f), f32 viewAngle (a),
- *   s16 xPos (x), s16 yPos (y), s16 zPos (z)
- * FORMAT
- *   Capital U is Unused
- *   ccrrffff aaaaaaaa xxxxyyyy zzzzUUUU
- *   size = 0x10
- */
-#define CS_CAM_FOCUS_POINT CS_CAM_AT
-#define CS_CAM_AT(continueFlag, roll, frame, viewAngle, xPos, yPos, zPos, unused) \
-    CMD_BBH(continueFlag, roll, frame), CMD_F(viewAngle), CMD_HH(xPos, yPos), CMD_HH(zPos, unused)
+#define CS_CAM_EYE_SPLINE(startFrame, endFrame) \
+    CS_CMD_CAM_EYE_SPLINE, CMD_HH(0x0001, startFrame), CMD_HH(endFrame, 0x0000)
 
 /**
- * ARGS
- *   s32 entries (e)
- * FORMAT
- *   00000003 eeeeeeee
- *   size = 0x8
+ * Declares a list of camera `at` points that form a spline.
+ * The points of the spline are interpolated over time to create smooth camera movements.
  */
-#define CS_MISC_LIST(entries) CS_CMD_MISC, CMD_W(entries)
+#define CS_CAM_AT_SPLINE(startFrame, endFrame) \
+    CS_CMD_CAM_AT_SPLINE, CMD_HH(0x0001, startFrame), CMD_HH(endFrame, 0x0000)
 
 /**
- * ARGS
- *   s16 unk (u), s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   Capital U is Unused
- *   uuuussss eeeeUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU
- *   size = 0x30
+ * Declares a list of camera `eye` points that form a spline.
+ * The points of the spline are interpolated over time to create smooth camera movements.
+ * The camera is moved relative to player.
  */
-#define CS_MISC(unk, startFrame, endFrame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7, unused8, unused9, unused10) \
-    CMD_HH(unk, startFrame), CMD_HH(endFrame, unused0), \
-    CMD_W(unused1), CMD_W(unused2), CMD_W(unused3), CMD_W(unused4), CMD_W(unused5), \
-    CMD_W(unused6), CMD_W(unused7), CMD_W(unused8), CMD_W(unused9), CMD_W(unused10)
+#define CS_CAM_EYE_SPLINE_REL_TO_PLAYER(startFrame, endFrame) \
+    CS_CMD_CAM_EYE_SPLINE_REL_TO_PLAYER, CMD_HH(0x0001, startFrame), CMD_HH(endFrame, 0x0000)
 
 /**
- * ARGS
- *   s32 entries (e)
- * FORMAT
- *   00000004 eeeeeeee
- *   size = 0x8
+ * Declares a list of camera `at` points that form a spline.
+ * The points of the spline are interpolated over time to create smooth camera movements.
+ * The camera is moved relative to player.
  */
-#define CS_LIGHTING_LIST(entries) CS_CMD_SET_LIGHTING, CMD_W(entries)
-
+#define CS_CAM_AT_SPLINE_REL_TO_PLAYER(startFrame, endFrame) \
+    CS_CMD_CAM_AT_SPLINE_REL_TO_PLAYER, CMD_HH(0x0001, startFrame), CMD_HH(endFrame, 0x0000)
 
 /**
- * ARGS
- *   s16 setting (m), s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   Capital U is Unused
- *   mmmmssss eeeeUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU 00000000 00000000 00000000
- *   size = 0x30
+ * Declares a single entry of camera `eye` data that will be set on the specified frame.
+ * Only the first camera point entry will be read, this is not processed as a list.
```

Comment:
```suggestion
 * Declares a single `eye` point that will be set on the specified frame, without any interpolation.
 * Should be followed by a single point, not a list.
```
idk

Trying to get the comment more consistent in terminology usage with its `CS_CAM_AT_SPLINE` counterpart
Also I don't think there needs to be that much emphasis on it not being followed by an arbitrary-length list

Reply:
i removed "without any interpolation" cause theres nothing that could even be interpolated against. It was necessary when I thought you could have multiple poitns, cause it could be mistaken that they will smoothly change between each other

Reply:
I mean, it could be interpolating towards the point from the current at/eye, it's not a rare occurrence in z_camera, but yeah it can probably be omitted

Reply:
yea guess that fair, ill add it back